Helper/Store.php

```diff
@@ -40,6 +40,22 @@ public function __construct(
         parent::__construct($context);
     }
 
+    /**
+     *  formats a single store into a more succinct and usable array
```

Comment:
I know this is super duper nitpicky but there's an extra whitespace at the beginning of this description and it should start with a capital letter.